marklogic-data-hub-central/ui/src/components/flows/new-flow-dialog/new-flow-dialog.module.scss

```diff
@@ -18,6 +18,7 @@
 .questionCircle {
     font-size: 12px;
     color: #7F86B5;
+    margin-left: 7px;
 }
 
 .questionCircle {
```

Comment:
@federicoRufrancosMarklogic  There are two `.questionCircle`. Can you remove the repeated one?